Directory.Build.props

```diff
@@ -9,7 +9,7 @@
     <PackageProjectUrl>https://github.com/qdrant/qdrant-dotnet</PackageProjectUrl>
     <PackageReleaseNotes>https://github.com/qdrant/qdrant-dotnet/releases</PackageReleaseNotes>
     <PackageTags>qdrant, database, vector, search</PackageTags>
-    <QdrantVersion>v1.8.0</QdrantVersion>
+    <QdrantVersion>dev</QdrantVersion>
```

Comment:
TODO: Switch to `v1.9.0` when out.

Reply:
Done in b333636bb45b986eb40641ce228cecfa5b8942fe